ui/desktop/src/App.tsx

```diff
@@ -986,12 +986,12 @@ export default function App() {
             if (error instanceof MalformedConfigError) {
               throw error;
             }
-            window.location.hash = '#/welcome';
-            window.history.replaceState({}, '', '#/welcome');
+            window.location.hash = '#/';
```

Comment:
without this it will never show the providerguard correctly - and just the wall of providers which we don't want. 

I am curious if it would be worth the effort to mock up/unit test this for App.tsx (or perhaps it is more an e2e test)

Reply:
(I have a unit test for it)